src/site/lab9.rst

```diff
@@ -11,15 +11,18 @@ Lab #9
     * I want people to get used to working together in the labs
     * Peer teaching and peer learning is super effective
 
+.. Note::
 
-Draw The Tree
-=============
-
+    We are simply playing with binary nodes in this lab and not a tree class, nor are we creating a tree class. We
+    are, however, still making a tree with these nodes. 
 
-Binary Node
-===========
 
+Draw The Tree
+=============
 
+1. Get a piece of paper
```

Comment:
any reason, specifically, paper? would paint (PINTA) work?

Reply:
Funnily enough, I don't think I ever saw a piece of paper in the last year or so; everyone just uses paint, etc., by default these days.

I'll leave it as "paper" since the _kids these days_ seem to take it as a more general thing than an actual physical paper.